### ISSUE TITLE
Add ACL support to Zookeeper (znode) module

### ISSUE THREAD

**Issue by tima** [ansible/ansible-modules-extras#829]
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

znode module
##### ANSIBLE VERSION

N/A
##### SUMMARY

Add the ACLs support in PR #310 from @jbowes to the module that @treyperry submitted in #811 that's been merged in to HEAD already.

**Comment by robynbergeron** [comment]
@tima Thanks for submitting this feature idea, and coordinating between the various zookeeper PRs to keep this particular feature going :) 

@treyperry -- as @tima mentioned, it looks like ACL support was included in #310 -- that one got closed by @jbowes since the functionality of this module was mostly covered by your PR (#811), but you could probably take a look at #310 for some inspiration.

Alternately -- @jbowes, since you're familiar with this -- perhaps you could give it a try -- just let the folks in this issue know so that you don't all go working on the same thing simultaneously :) 

Thanks to all of you for your work and care in extras -- much appreciated! And now, for some exciting standard cut and paste triage info :) Cheers!

*\* 

@treyperry -- Pinging you to bring this feature request for your attention. Please note that you are under no particular obligation to add this feature.

We encourage all Ansible community members to submit pull requests to satisfy feature requests. Consult the module guidelines (http://docs.ansible.com/ansible/developing_modules.html) for more information.

**Comment by ghost** [comment]
Has this feature been implemented yet? I am looking to use it.

**Comment by tima** [comment]
@sarlindo: To my knowledge no. Feel free to submit a PR for it. I just recorded it during code reviews and had hoped @jbowes or @treyperry would get to it since it's highly unlikely I would. (I haven't worked with Zookeeper in a couple of years and have a lot of other things on my desk to handle.)

**Comment by ansibot** [comment]
@tima: Thanks for taking the time to open this issue, but it was filed under the wrong github repository. Please create a new issue at https://github.com/ansible/ansible-modules-core/issues
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@treyperry, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@treyperry, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@treyperry, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/29216
